orchestrator/dags/utils.py

```diff
@@ -1,12 +1,38 @@
 import settings
+import requests
 
 
-def generate_api_warszawa_url(api_key: str, resource_id: str, **kwargs):
-    """
-    Returns url to Warsaw Public Transport API with given parameters as query strings
-    """
-    url = settings.API_WARSZAWA_URL
-    url += f'?apikey={api_key}&resource_id={resource_id}'
-    for k, v in kwargs.items():
-        url += f'&{k}={v}'
-    return url
+class ApiWarsawUrl:
+    BASE_URL = 'https://api.um.warszawa.pl/'
+    ENDPOINT_DBSTORE = 'api/action/dbstore_get/'
+    ENDPOINT_BUSESTRAMS = 'api/action/busestrams_get/'
+
+    def __init__(self, apikey):
+        self._client = requests.session()
+        self._apikey = apikey
+
+    def _get(self, url, params=None):
+        if params is None:
+            params = {}
+        default_params = {"apikey": self._apikey}
+        default_params.update(params)
+        response = self._client.get(
+            f"{ApiWarsawUrl.BASE_URL}{url}", params=default_params
+        )
+        response.raise_for_status()
+        return response.status_code, response.json()
```

Comment:
```suggestion
        response.raise_for_status()
        return response.json()
```
If there is a call for `raise_for_status`, there is rather no need to return the status code (uneless we really need to handle different 2xx statuses in the executed code). Take a look at the [source of `raise_for_status`](https://requests.readthedocs.io/en/latest/_modules/requests/models/#Response.raise_for_status)